### PR TITLE
Homepage updates

### PR DESCRIPTION
**Pull request by mbouzi**
- mock - https://www.figma.com/file/0saZ6iBgSsn8BQJUP8k5x1Pu/Storj.io?node-id=2256%3A724
- added hover shadows to hero sections (host a node)
- fixed responsiveness of "Introducing Tardigrade" section
- new background for hero
- addition of buttons and logos in hero

### CODE REVIEW

File: layouts/partials/home/Hero.html [hunk 1]
```diff
@@ -23,52 +23,47 @@ <h1 class="hero-heading col-xs-12">{{.Params.subHeading}}</h1>
...
     {{end}}
     <div class="hero-sections row">
-      <div class="hero-host col-lg-6">
-        <div>
-          <div class="spacer60"></div>
-          <div class="row">
-            <div class="col-lg-12">
-              <img style="margin-bottom:15px;" src="/img/storj-ico-node.svg"/>
-            </div>
+      <div onclick="location.href='/sign-up-node-operator';" style="cursor: pointer;" class="hero-host col-lg-6">
```
Comment (VitaliiShpital) [inline layouts/partials/home/Hero.html]: we should avoid inline styles

File: layouts/partials/home/Hero.html [hunk 2]
```diff
@@ -23,52 +23,47 @@ <h1 class="hero-heading col-xs-12">{{.Params.subHeading}}</h1>
...
         </div>
       </div>
-      <div class="hero-cs col-lg-6 careers-grey">
-        <div>
-          <div class="spacer50"></div>
-          <div class="row">
-            <div class="col-lg-12">
-              <img src="/img/storj-ico-cloud-storage.svg"/>
-            </div>
+      <div onclick="location.href='https://tardigrade.io/';" style="cursor: pointer;" class="hero-cs col-lg-6 careers-grey">
```
Comment (VitaliiShpital) [inline layouts/partials/home/Hero.html]: same about inline styles

File: layouts/partials/home/Hero.html [hunk 3]
```diff
@@ -23,52 +23,47 @@ <h1 class="hero-heading col-xs-12">{{.Params.subHeading}}</h1>
...
+				<div class="spacer60"></div>
+        <div class="row">
+          <div class="col-lg-12">
+            <img src="/img/tardigrade-hero-logo.svg"/>
```
Comment (VitaliiShpital) [inline layouts/partials/home/Hero.html]: we should add alt="tardigrade hero logo" here

File: layouts/partials/home/Hero.html [hunk 4]
```diff
@@ -23,52 +23,47 @@ <h1 class="hero-heading col-xs-12">{{.Params.subHeading}}</h1>
...
-          <div class="spacer60"></div>
-          <div class="row">
-            <div class="col-lg-12">
-              <img style="margin-bottom:15px;" src="/img/storj-ico-node.svg"/>
-            </div>
+      <div onclick="location.href='/sign-up-node-operator';" style="cursor: pointer;" class="hero-host col-lg-6">
+        <div class="spacer50"></div>
+        <div class="row">
+          <div class="col-lg-12">
+            <img src="/img/storj-hero-logo.svg"/>
```
Comment (VitaliiShpital) [inline layouts/partials/home/Hero.html]: alt="Storj hero logo"

File: layouts/partials/home/Hero.html [hunk 5]
```diff
@@ -23,52 +23,47 @@ <h1 class="hero-heading col-xs-12">{{.Params.subHeading}}</h1>
...
+            <p class="hero-desc">Upgrade your storage layer to the world’s first S3-compatible decentralized cloud storage solution. Tardigrade is 100% secure, 20% faster, and half the cost.</p>
+            <div class="spacer15"></div>
+              <p class="hero-sections-cta">Join Tardigrade</p>
+              <!-- <img src="/img/carrot-right.svg" height="16" width="16"> -->
           </div>
         </div>
       </div>
     </div>
     <div class="spacer125"></div>
+		<img class="globe-background" src="/img/globe.png">
```
Comment (VitaliiShpital) [inline layouts/partials/home/Hero.html]: please add alts to img tag

File: static/css/style.css [hunk 6]
```diff
@@ -4198,9 +4279,9 @@ color: #FFFFFF;
   }
 }
 
-.introducing .container {
+/* .introducing .container {
```
Comment (VitaliiShpital) [inline static/css/style.css]: should we delete this block of code?

**Comment by egonelbre** [comment]
Please ensure that no image loaded is larger than 250KB.

**Comment by egonelbre** [comment]
PS: the 250kb doesn't have to happen in this PR, you can do follow-up PR-s if that's nicer.

**Pull request merged**